src/main/java/edu/cornell/kfs/fp/batch/xml/DisbursementVoucherPaymentInfomrationXml.java

```diff
@@ -0,0 +1,334 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import java.util.Date;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.sys.xmladapters.KualiDecimalXmlAdapter;
+import edu.cornell.kfs.sys.xmladapters.StringToJavaDateAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "PaymentInformation", namespace = StringUtils.EMPTY)
```

Comment:
Should this be payment_information?

Reply:
no, that tag is on DisbursementVoucherDetailXml, where this attribute is used, this root element just needs to be unique.  I will go through the new ojbects, and check for consistency